modules/gdscript/gdscript_compiler.h

```diff
@@ -38,9 +38,8 @@
 class GDScriptCompiler {
 
 	const GDScriptParser *parser;
-	Map<StringName, Ref<GDScript> > class_map;
-	Set<StringName> parsed_classes;
-	Set<StringName> parsing_classes;
+	Set<GDScript*> parsed_classes;
+	Set<GDScript*> parsing_classes;
```

Comment:
Style issue: https://travis-ci.org/godotengine/godot/jobs/507240380

Reply:
Hm, seems like the clang-format plugin wasn't configured to work on .c/.h files. Thanks!